package.json

```diff
@@ -109,13 +110,20 @@
     "recursive-readdir": "2.2.1",
     "replace-in-file": "3.0.0",
     "rheostat": "2.1.1",
+    "rollup": "0.51.3",
+    "rollup-plugin-babel": "3.0.2",
+    "rollup-plugin-commonjs": "8.2.6",
+    "rollup-plugin-filesize": "1.4.2",
+    "rollup-plugin-node-globals": "1.1.0",
+    "rollup-plugin-node-resolve": "3.0.0",
+    "rollup-plugin-replace": "2.0.0",
+    "rollup-plugin-uglify": "2.0.1",
     "sass-loader": "6.0.6",
     "stat-mode": "0.2.2",
     "storybook-addon-a11y": "3.1.9",
     "storybook-addon-jsx": "5.0.0",
     "string": "3.3.3",
     "style-loader": "0.19.0",
-    "uglify-js": "2.7.5",
     "webpack": "3.8.1",
     "webpack-bundle-analyzer": "2.9.1",
```

Comment:
same for those I guess, cc https://github.com/algolia/react-instantsearch/pull/603/files#r151405932

Reply:
Actually it's referenced inside the documentation pipeline so I don't touch it.

https://github.com/algolia/react-instantsearch/blob/master/docgen/webpack.config.build.babel.js#L5